shared-ui/components/meet-the-team/meet-all-core/MeetAllCore.styles.ts

```diff
@@ -0,0 +1,53 @@
+import styled from 'styled-components';
+import { P, H3 } from '../../../style/typography';
+import { colors } from '../../../style/colors';
+import { max } from '../../../lib/responsive';
+
+const StyledMeetCoreContainer = styled.div`
+  display: flex;
+  margin: auto;
+  padding: 8em;
+  @media ${max.tabletLg} {
+    padding: 4em;
+  }
+`;
+
+const StyledOurTeamCore = styled(P)`
+  color: ${colors.WHITE};
+  text-align: left;
+  padding-top: 3.5em;
```

Comment:
Can we only apply this at larger screensizes? I don't think we need this much space on mobile

<img width="781" alt="Screenshot 2022-11-19 at 4 26 48 PM" src="https://user-images.githubusercontent.com/59738880/202872222-50a0cff8-cffd-44af-bff2-f2f68728e028.png">


like desktop or tabletLg? 

```
  @media ${min.desktop} {
    padding-top: 3.5em;
  }
```